lib/nexpose/silo_profile.rb

```diff
@@ -107,7 +107,7 @@ def save(connection)
       begin
         update(connection)
       rescue APIError => error
-        raise error unless (error.message =~ /A silo profile .* does not exist./)
+        raise error unless (error.message =~ /The silo profile .* does not exist/)
```

Comment:
Some day we'll have a fully RESTful JSON API to replace this that returns appropriate status codes instead of messages that must be parsed. :smiley:

I am not too concerned about flexibility since this message really shouldn't change again (or if it does the above should come true first).